kale/embed/__init__.py

```diff
@@ -0,0 +1,2 @@
+# from ._mpca import MPCA
+from ._mpca_torch import MPCA
```

Comment:
add some references on the benefits/preference of doing so will help the understanding.